app/assets/stylesheets/users.scss

```
@@ -0,0 +1,182 @@
+$dark: #262626;
+$grey: #999;
+$grey-border-t: #efefef;
+$grey-border: #dbdbdb;
+$color-website: #003569;
+$color-photo: #3897f0;
+
+.profile .avatar{
+  .wrapper{
+    width: 140px;
+    height: 140px;
+    margin: 40px auto;
+  }
+  .wrapper img{
+    width: 100%;
+    height: 100%;
+    border-radius: 50%;
+  }
+}
+
+.profile .basic-info{
+  padding: 40px 0;
+  .user-wrapper{
+    display: flex;
+  }
+  .username{
+    font-size: 30px;
+    line-height: 40px;
+    font-weight: 200;
+  }
+  .edit-profile{
+    margin-left: 20px;
+    margin-top: 5px;
+  }
+  .edit-profile button{
+    background-color: transparent;
+    border-radius: 5px;
+    border: 1px solid $grey-border;
+    color: $dark;
+    font-size: 14px;
+    padding: 4px 10px;
+    font-weight: 500;
+  }
+  ul.posts{
+    padding-left: 0;
+    > li{
+      display: inline;
+      padding-right: 40px;
+      font-size: 15px;
+      span{
+        color: $dark;
+        font-weight: 600;
+      }
+    }
+  }
+  .name{
+    font-size: 15px;
+    font-weight: 600;
+  }
+  .website{
+    color: $color-website;
+    text-decoration: none;
+    font-size: 15px;
+    font-weight: 600;
+  }
+}
+
+.user-tabs{
+  border-top: 1px solid $grey-border-t;
+  display: flex;
+  justify-content: center;
+
+  a.tab{
+    height: 35px;
+    margin-right: 50px;
+    line-height: 45px;
+    color: $grey;
+    font-size: 12px;
+    font-weight: 500;
+    text-align: center;
+    i{
+      padding-right: 1px;
+    }
+    &:hover{
+      text-decoration: none;
+    }
+  }
+  a.active{
+    border-top: 1px solid $dark;
+    color: $dark;
+  }
+}
+
+.user-images{
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: flex-start;
+  margin: 0 20px;
+
+  .wrapper{
+    width: 280px;
+    height: 280px;
+    margin: 15px;
+
+    img{
+      width: 100%;
+      border-radius: 4%;
+      width: 278px;
+      height: 278px;
+      object-fit: cover;
+      border: 1px solid $grey-border-t;
+    }
+  }
+}
+
+/*  Edit Profile page */
+.user-edit-page{
+  display: flex;
+  margin-top: 60px;
+  min-height: 500px;
+
+  .actions{
+    width: 220px;
+    border: 1px solid $grey-border;
+
+    .nav-link{
+      font-size: 14px;
+      background-color: white;
+      color: black;
+      border-radius: 0;
+      padding: 12px 0 12px 30px;
```

css shorthand này ông ơi
``
```suggestion
      padding: 12px 0 30px;
```

---

Done nhá :)